### PR TITLE
Adds retrying to JSON parsing for FRED data

### PR DESCRIPTION
**Pull request by gsalaz98**
This change is made to improve stability of downloading/processing the data.

### CODE REVIEW

File: process.ipynb [hunk 1]
```diff
@@ -36,6 +24,7 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "ea477d94",
```
Comment (Martin-Molinero) [inline process.ipynb:27]: Wonder what's `id`?

File: process.ipynb [hunk 2]
```diff
@@ -639,21 +628,32 @@
...
+    "            data = full_response.loc[:, ['date', 'value']]\n",
+    "            data['value'] = pd.to_numeric(full_response.value, errors='coerce')\n",
+    "            data = data.set_index(pd.to_datetime(data.date, errors='coerce'), drop=True)\n",
+    "            data.drop('date', axis=1, inplace=True)\n",
+    "            \n",
+    "            return data.dropna()\n",
+    "        except Exception as e:\n",
+    "            ex = e\n",
+    "            print(f'Failed to convert JSON data to DataFrame from URL: {url} - retrying')\n",
+    "            sleep(1)\n",
```
Comment (Martin-Molinero) [inline process.ipynb:653]: Thinking if it should raise the exception after the retry else it wont fail?

File: process.ipynb (hunk 1 above)
Comment (gsalaz98) [reply process.ipynb:27]: https://jupyter.org/enhancement-proposals/62-cell-id/cell-id.html

Seems it's just an id for the cell, it doesn't do anything other than make it unique

File: process.ipynb (hunk 2 above)
Comment (gsalaz98) [reply process.ipynb:653]: Usually it's only one data source that causes the whole thing to fail, but yes we can make it throw :+1:

**Review by Martin-Molinero — APPROVED** [review]

**Pull request merged**